genai-perf/docs/goodput_tutorial.md

```diff
@@ -0,0 +1,439 @@
+<!--
```

Comment:
The tutorial needs to be further refined, the main purpose here is to show goodput functionality for LLM, Embeddings and Rankings under three conditions: without goodput-constraints, with valid goodput constraints, and with invalid goodput constraints.